content/news/028/index.md

```diff
@@ -84,6 +84,44 @@ out [this form][gamedev-meetup-form].
 
 ## Game Updates
 
+### Shroom Kingdom
+
+![Shroom Kingdom game preview](shroom-kingdom.gif)
+_Starting the game and placing some blocks_
+
+Shroom Kingdom ([GitHub][shrm-github], [Discord][shrm-discord],
+[Twitter][shrm-twitter]) is an upcoming play-to-earn video game built with web
+technologies running on the [NEAR Blockchain][near-blockchain].
+
+Currently a prototype of the game is in development.
+An asset file, that has been extracted from Super Mario Maker 2 game files with an
+[external program](asset-extractor), is loaded inside a browser to start the game.
+The game is written with the [Bevy game engine](bevy)
+and the [Rapier physics engine](rapier) and is compiled to WebAssembly.
+The prototype is still very basic, but you can already place and remove blocks
+and have a feeling about the physics recreation of the original games.
+
+In the past months there also has been the SHRM token launch and a
+[token airdrop](shrm-airdrop) for NEAR early adopters. The [Shroom Kingdom DAO](shrm-dao)
```

Comment:
Links need to be square brackets, not circular ones

Reply:
which link do you mean? I think I made them all in square brackets here

Reply:
[Rapier physics engine](rapier) should be [Rapier physics engine][rapier]

Reply:
Oh yeah sorry. I was so used to writing links with regular brackets